package-lock.json

```diff
@@ -1,18 +1,389 @@
 {
   "name": "wellcome-design-system",
-  "version": "0.0.7",
+  "version": "0.0.8",
```

Comment:
It seems that the` package-lock.json` needs to be updated, as we are currently on version **0.0.17**.

Reply:
hmmmm – yeah that's strange. I don't think it's standard to be changing things in package-lock yourself because it's generated) but I'll see if I can work out what's happening